bleedbluedocs/docs.css

```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200&display=swap');
+@import url(../components/css/bleedblue.css);
+
+
+.page-layout{
+    margin:0;
+    padding: 0;
+    gap:0;
+    height: 100vh;
+    display:grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    grid-template-areas: 
+    "header header header header header"
+    "aside   main   main   main   main"
+    "footer footer footer footer footer";
+}
+
+
+.page-header {
+    grid-area: header;
+    z-index: 1;
+}
+.page-content {
+    grid-area: main;
+}
+.page-footer{
+    grid-area: footer;
+    z-index: 1;
+}
+
+
+.sidebar {
+    grid-area: aside;
+    position:relative;
+}
+
+
+.sidenav{
+    position: sticky;
+    overflow-y: auto;
+    top: 0;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    gap:1rem;
+    color: var(--bb-dark-color);
+    padding-left: 2rem;
+
+}
+
+
+.sidenav a {
+    color: var(--bb-dark-color);
+}
+
+
+
+@media only screen and (max-width: 820px) {
+
+    .page-layout{
+        display:grid;
+        grid-template-columns: 1fr;
+        grid-template-areas: 
+        "header header header header header"
+        "main   main   main   main   main"
+        "footer footer footer footer footer";
+    }
+    .page-header {
+        grid-area: header;
+        z-index: 1;
+    }
+    .page-content {
+        grid-area: main;
+    }
+    .page-footer{
+        grid-area: footer;
+        z-index: 1;
+    }
+    
+    
+    .sidebar {
+        display: none;
+    }
+}
+
+
+
+/* component container styling */
+
```

Comment:
Formatting suggestion : The spacing between lines seems to differ, it would be nice if the same pattern is followed throughout for better readability.

Reply:
Thanks for the suggestion, I'll look into that.